api/__tests__/events.test.js

```diff
@@ -6,7 +6,7 @@ const nock = require('nock')
 
 axios.defaults.adapter = require('axios/lib/adapters/http')
 
-const scope = nock('http://localhost:5000')
+const scopeEvents = nock('http://localhost:3000')
     .get('/airtable/events')
```

Comment:
Not a biggie, but wonder if it might be useful to make all the events endpoints begin simply `/events` rather than `/airtable/events`?  Then if we changed the source of the events data later so it's no longer coming from AirTable, the URL wouldn't need to be updated here or in the app.  Also maybe it's good for anything consuming this endpoint not to have to care where the data's coming from?  All they need to know is what thing (e.g. events) they want.  Totally appreciate there may be other good reasons why you did this though.

Reply:
That is a possibility, I added the airtable incase something other than evens were added